src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -714,5 +711,26 @@ public static long parseLong(final String s) {
 		}
 		return lastIndex;
 	}
+
+	/**
+	 * Removes the first n elements from a Collection that matches the filter
+	 *
+	 * @param coll the {@link Collection} to remove from.
+	 * @param n the amount of elements to remove
+	 * @param filter the {@link Predicate} / filter that will be checked to remove the elements
+	 * @return amount of elements that matched the filter and have been removed.
+	 */
+	public <E> int removeFirstIf(Collection<E> coll, int n, Predicate<E> filter) {
```

Comment:
Should be static and in CollectionUtils, also you're not using this method, so I'm not sure if we should have it at all

Reply:
Kept it and moved to `CollectionUtils` as there are some unused methods in that class as well and this may be needed someday later